index.html

```diff
@@ -5,8 +5,42 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Henry Cadena Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <h1>Henry Cadena Portfolio Mobile First</h1>
+    <header class="container-header">
+      <h3 class="logo">Henry Cadena</h3>
+      <div class="container-burger">
+        <label class="burger1"></label>
+        <label class="burger2"></label>
+        <label class="burger3"></label>
+      </div>
```

Comment:
- [ ] To follow the HTML5 structure, it is recommended to include a `<nav>` tag inside the `<header>` tag. Wrapping the navigation section with the `<nav>` tag provides a more semantically meaningful structure to your HTML code. 
- [ ] This helps in organizing and distinguishing the navigation section from other content within the header, aligning with the best practices of HTML5.🤔🤔
- [ ] For more information please [check it](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure)